scripts/publish_gh_release_notes.py

```diff
@@ -61,12 +61,6 @@ def parse_changelog(tag_name: Text) -> Text:
     return "\n".join(version_lines[2:]).strip()
 
 
-def convert_rst_to_md(text):
-    return pypandoc.convert_text(
```

Comment:
can you remove the `pypandoc` import from the top of the file? and remove the dependency in the CI?